packages/react/src/navigation-menu/trigger/NavigationMenuTrigger.tsx

```diff
@@ -91,10 +97,32 @@ export const NavigationMenuTrigger = React.forwardRef(function NavigationMenuTri
   const [stickIfOpen, setStickIfOpen] = React.useState(true);
   const [pointerType, setPointerType] = React.useState<'mouse' | 'touch' | 'pen' | ''>('');
 
+  const allowFocusRef = React.useRef(false);
+  const prevSizeRef = React.useRef(DEFAULT_SIZE);
+  const abortControllerRef = React.useRef(DEFAULT_ABORT_CONTROLLER);
+
   const isActiveItem = open && value === itemValue;
   const isActiveItemRef = useLatestRef(isActiveItem);
+  const interactionsEnabled = positionerElement ? true : !value;
 
-  const allowFocusRef = React.useRef(false);
+  const runOnceAnimationsFinish = useAnimationsFinished(popupElement);
+
+  if (!open && pointerType !== '') {
+    setPointerType('');
+  }
+
+  React.useEffect(() => {
+    abortControllerRef.current.abort();
+  }, [isActiveItem]);
```

Comment:
To give you an example regarding the naming of this variable, while reviewing this PR I had no idea what this effect was about until I found where `abortControllerRef` was used below. Ideally the naming should make it clear what it's used for, without having to look for what the code does.